website/content/en/preview/getting-started/scripts/run-all-steps.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+echo "Step 0"
```

Comment:
Should we just for-loop in numerical order inside of here so that if others are added this file doesn't need to be updated?